### PR TITLE
add chalk, polonius, datafrog, rustc-guide

### PR DESCRIPTION
**Pull request by nikomatsakis**
r? @pietroalbini

### CODE REVIEW

**Comment by nikomatsakis** [comment]
@mark-i-m I added you as a "default PR assignee" for rustc-guide, hope that's ok :)

**Comment by mark-i-m** [comment]
:+1: Thanks for the heads up :smiley:

**Comment by mark-i-m** [comment]
Concretely, what does that mean? If a PR is made that changes the rust-lang/rust submodule for the guide I will be assigned?

**Comment by pietroalbini** [comment]
Nah, this is for PRs on the rustc-guide repo itself.

File: highfive/configs/rust-lang/rust-guide.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (davidalber) [inline highfive/configs/rust-lang/rust-guide.json]: Should this file be rustc-guide.json (note the "c" in rustc)?

Comment (pietroalbini) [reply highfive/configs/rust-lang/rust-guide.json]: Yep, woops, already pushed a fix.

**Pull request merged**